subsys/bluetooth/controller/hci_internal.c

```diff
@@ -1201,6 +1201,8 @@ static uint8_t vs_cmd_put(uint8_t const * const cmd,
 		return sdc_hci_cmd_vs_conn_event_extend((void *)cmd_params);
 	case SDC_HCI_OPCODE_CMD_VS_QOS_CONN_EVENT_REPORT_ENABLE:
 		return sdc_hci_cmd_vs_qos_conn_event_report_enable((void *)cmd_params);
+	case SDC_HCI_OPCODE_CMD_VS_QOS_CHANNEL_SURVEY_ENABLE:
```

Comment:
Please add some ifdefs around here to ensure that the feature is not linked in when not enabled